server/routes/featureRoutes.js

```diff
@@ -42,12 +42,24 @@ var addFeature = function(req, res){
 var updateFeatureToggle = function(req, res){
     var applicationName = req.params.applicationName;
     var featureName = req.params.featureName;
-    var value = req.body.value;
 
-    feature.updateFeatureToggle(applicationName, featureName, value, req, function(err){
-        if (err) throw err;
-        res.send(200);
-    });
+    if(req.body.newFeatureDescription) {
```

Comment:
So here we are doing two things with the same route, depending on the body? Not sure I like this - suggest we have two different routes maybe?